src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/MovementResourceIntTest_createExternalMovement.java

```diff
@@ -0,0 +1,119 @@
+package uk.gov.justice.hmpps.prison.api.resource.impl;
+
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.reactive.server.WebTestClient.ResponseSpec;
+import uk.gov.justice.hmpps.prison.api.model.CreateExternalMovement;
+import uk.gov.justice.hmpps.prison.repository.jpa.model.MovementDirection;
+import uk.gov.justice.hmpps.prison.service.EntityNotFoundException;
+import uk.gov.justice.hmpps.prison.service.MovementsService;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+
+@SuppressWarnings("ALL")
+public class MovementResourceIntTest_createExternalMovement extends ResourceTest {
```

Comment:
I've created a new class here as the old was using MockMvc which doesn't play nicely anymore with spring boot 3.  Tests have all been copied over.

Reply:
had I realised that github would treat it as a brand new file I'd have written it from scratch in kotlin instead 😁